src/omnisharp/OmnisharpDownloader.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fs from 'fs';
+
+export class OmnisharpDownloader {
```

Comment:
I'm confused by this name since this class doesn't actually download anything at all.